src/cli.js

```diff
@@ -29,6 +29,9 @@ var logger = require('cordova-common').CordovaLogger.get();
 var Configstore = require('configstore');
 var conf = new Configstore(pkg.name + '-config');
 var editor = require('editor');
+const semver = require('semver');
+
+const NODE_VERSION_REQUIREMENT = '>=8';
```

Comment:
Would be even better to use the value from `engines.node` https://github.com/apache/cordova-cli/blob/7c302ad4d610dda63bff7cfab873c972793adbef/package.json#L8 After it has been updated of course.

Reply:
Yes, that is the plan. Once we start preparing the next major release and the `package.json` `engine.node` is updated to `>=10`, we should be able to pull from there and is preferred.